test/gurka/test_gtfs.cc

```diff
@@ -152,6 +152,15 @@ TEST(GtfsExample, WriteGtfs) {
     .stop_timezone = "America/Toronto", .wheelchair_boarding = "1",
   };
   feed.add_stop(thirdStop);
+  struct gtfs::Stop thirdStopPlatform {
+    .stop_id = stopThreeID + "_walk_the_plank_pal", .stop_name = gtfs::Text("THIRD STOP"),
```

Comment:
this makes sure to test the case that we can find the graph id for platforms who have actual ids rather than being generated (like the second stations platform above)